frontend/src/components/about/AboutUs.js

```diff
@@ -85,7 +82,6 @@ class AboutUs extends React.Component {
                                 </div>
                                 <div class="card-body text-center">
                                     <h5 class="card-title">Aasma</h5>
```

Comment:
I don't know how long there wont be corresponding pictures of Abed and Aasma, maybe take advantage of that this branch is opened and put a default one? like avatar facebook type maybe?... Just giving ideas here.

Reply:
Can always do later via another PR - shouldn't cause any issues. Can get M to take pics of them tomorrow if none by then.